pingcap/tiflash
# clear old range before apply snapshot

### What problem does this PR solve?

Issue Number: close #4414

Problem Summary:

### What is changed and how it works?

### Check List

Tests 

1. load data into tidb cluster with multiple tiflash nodes by import csv file and set tiflash replica to 1;
2. set tiflash replica to 0 and see whether the data can be reclaimed;

Side effects


Documentation


### Release note



```release-note
Avoid left data on tiflash node after set tiflash replica to 0
```
